dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java

```diff
@@ -216,9 +227,15 @@ public void testQueryWithDifferentAssembly() throws Exception {
 
     @Test
     public void testQueryWithDifferentAssemblyType() throws Exception {
-        reader = buildReader(CHICKEN_ASSEMBLY_5, Collections.singletonList(NON_NUCLEAR), PAGE_SIZE);
-
+        reader = buildReader(BATCH, CHICKEN_ASSEMBLY_5, Collections.singletonList(NON_NUCLEAR), PAGE_SIZE);
         List<SubSnpCoreFields> list = readAll(reader);
         assertEquals(0, list.size());
     }
+
+    @Test
+    public void testQueryWithDifferentBatch() throws Exception {
```

Comment:
Wouldn't it be more interesting to test a non-existing batch?

Reply:
I'll put both